### PR TITLE
Update requests version, fix wellness object

### PR DESCRIPTION
**Pull request by freekode**
As the PR will be merged, please make a new release of the package on pypi

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-requests==2.25.1
+requests>=2.28.1
```
Comment (rday) [inline requirements.txt:1]: Any reason for `>=` and not locking to a version?

File: src/intervalsicu/api/wellness.py [hunk 2]
```diff
@@ -1,36 +1,54 @@
 class Wellness(dict):
     fields = [
-        'diastolic',
-        'fatigue',
+        'id',
```
Comment (rday) [inline src/intervalsicu/api/wellness.py:3]: Thanks! I have a local version which doesn't lock in the fields. There are a couple structures that change based on available data in the server, and this method always fails :(  I'll push that change out soon.

**Review by rday — APPROVED** [review]
I'll merge this in and update the package later tonight.

File: requirements.txt (hunk 1 above)
Comment (freekode) [reply requirements.txt:1]: Just for better compatibility with other code, where the lib will be used.

**Pull request merged**